ReactiveCocoa/Swift/Signal.swift

```diff
@@ -311,11 +311,11 @@ extension SignalType where Value: SignalProducerType, Error == Value.Error {
 	/// emitted from `signal` complete.
 	@warn_unused_result(message="Did you forget to call `observe` on the signal?")
 	private func concat() -> Signal<Value.Value, Error> {
-		return Signal<Value.Value, Error> { [signal = self.signal] observer in
+		return Signal<Value.Value, Error> { observer in
```

Comment:
I'm not sure about this - doesn't this now capture `self` with a strong reference (thus leading to a potential strong reference cycle), whereas it previously had captured only `signal` and not `self`?